src/main/java/wooteco/subway/domain/DiscountByAgePolicy.java

```diff
@@ -0,0 +1,44 @@
+package wooteco.subway.domain;
+
+import java.util.Arrays;
+import java.util.function.Function;
+import java.util.function.Predicate;
+
+public enum DiscountByAgePolicy {
+
+    BABY_POLICY(
+            (fare) -> 0,
+            (age) -> 1 <= age && age < 6
+    ),
+    KIDS_POLICY(
+            (fare) -> (int) Math.ceil((fare - 350) * 0.5),
+            (age) -> 6 <= age && age < 13
+    ),
+    TEENAGER_POLICY(
+            (fare) -> (int) Math.ceil((fare - 350) * 0.8),
+            (age) -> 13 <= age && age < 19
+    ),
+    ADULT_POLICY(
+            (fare) -> fare,
+            (age) -> 19 <= age
+    );
```

Comment:
저는 보통 변수를 왼쪽에 놓고 사용해요..!

왼쪽에서 선언된 변수를 보고 age >= 6을 읽고 age < 13에서 머리속에서 자연스레 age를 생략하고 < 13을 읽는 방식으로 읽는게 자연스럽더라구요 😄
변수를 주로 왼쪽에 선언하는 방식으로 구현해서 익숙하더라구요 ㅎ_ㅎ..

라라가 익숙하신 방식으로 작성하시면 될 것 같아요 !

```suggestion
    BABY_POLICY(
            (fare) -> 0,
            (age) -> age >= 1  && age < 6
    ),
    KIDS_POLICY(
            (fare) -> (int) Math.ceil((fare - 350) * 0.5),
            (age) -> age >= 6  && age < 13
    ),
    TEENAGER_POLICY(
            (fare) -> (int) Math.ceil((fare - 350) * 0.8),
            (age) -> age >= 13 && age < 19
    ),
    ADULT_POLICY(
            (fare) -> fare,
            (age) -> age >= 19
    );
```

Reply:
변수를 왼쪽에 두니 age 가 1보다 작고 라고 매끄럽게 읽히는 것 같네요!! 😆👍🏻

Reply:
엇.. 큰일이네요 🥺  age가 1보다 크다라고 읽혀야 하는데요..! ㅎㅎㅎㅎ 오타인걸로 알고 넘어갈게요 !